src/index.html

```diff
@@ -0,0 +1,217 @@
+                        <button type="button" class="btn-reserve">Reservation</button>
+                    </div>
+                </div>
+
+                <div class="moovie-item">
+                    <div class="moovie-item-image-container">
+                        <div class="moovie-item-picture">
+                        </div>
+                        <div class="moovie-item-image-description">
+                            <h4 class="item-name">Name</h4>
+                            <div class="moovie-item-likes">
+                                <p class="heart"><i class="fa-regular fa-heart" id="likeIcon"></i></p>
+                                <input type="hidden" name="idmoovie" value="" class="moovieId">
+                                  <p class="likes-number"></p>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="moovie-item-options">
+                        <button type="button" class="btnComment">Comments</button>
+                        <button type="button" class="btn-reserve">Reservation</button>
+                    </div>
+                </div>
+                <div class="moovie-item">
+                    <div class="moovie-item-image-container">
+                        <div class="moovie-item-picture">
+                        </div>
+                        <div class="moovie-item-image-description">
+                            <h4 class="item-name">Name</h4>
+                            <div class="moovie-item-likes">
+                                <p class="heart"><i class="fa-regular fa-heart" id="likeIcon"></i></p>
+                                <input type="hidden" name="idmoovie" value="" class="moovieId">
+                                  <p class="likes-number"></p>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="moovie-item-options">
+                        <button type="button" class="btnComment">Comments</button>
+                        <button type="button" class="btn-reserve">Reservation</button>
+                    </div>
+                </div>
+                <div class="moovie-item">
+                    <div class="moovie-item-image-container">
+                        <div class="moovie-item-picture">
+                        </div>
+                        <div class="moovie-item-image-description">
+                            <h4 class="item-name">Name</h4>
+                            <div class="moovie-item-likes">
+                                <p class="heart"><i class="fa-regular fa-heart" id="likeIcon"></i></p>
+                                <input type="hidden" name="idmoovie" value="" class="moovieId">
+                                  <p class="likes-number"></p>
+                            </div>
+                        </div>
+                    </div>
```

Comment:
This whole section should be generated dynamically using JS. However, in your case, you have written the HTML manually which makes the concept of a dynamic website pointless. Imagine if you have 20 or 30 movies. The HTML would be very long in that case, which is not a point of this module. Kindly remove this hard-coded HTML and use JS to generate the code.